src/spring/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.26.0'
+VERSION = '1.26.1'
```

Comment:
Due to the impact of CCOA, if you want to merge this PR in advance, please do not upgrade the extension version for now